### PR TITLE
Spike/432 EDC helm chart

### PR DESCRIPTION
**Pull request by ouphi**
Added EDC helm chart.

### CODE REVIEW

File: helm/templates/statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            httpGet:
+              port: http
+              path: /api/check/liveness
+
+          startupProbe:
+            initialDelaySeconds: 10
+            failureThreshold: 7
+            httpGet:
+              port: http
+              path: /api/check/startup
```
Comment (algattik) [inline helm/templates/statefulset.yaml]:
```suggestion
              path: /api/check/readiness
```
See discussion at https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/412

File: helm/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/use-regex: "true"
```
Comment (algattik) [inline helm/templates/ingress.yaml]: Helm charts should either have a properly configurable ingress controller (where ingress class and annotations are passed as templated values), or not have one at all.

File: helm/templates/statefulset.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      containers:
+        - name: edc
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          ports:
+            - containerPort: {{ .Values.container.port }}
+              name: http
+
+          readinessProbe:
+            initialDelaySeconds: 1
```
Comment (algattik) [inline helm/templates/statefulset.yaml:36]: is there a particular reason to override the default initialDelaySeconds values?

File: helm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+image:
+  repository: <container-registry-repository>
+  tag: <image-tag>
+  pullPolicy: Always # Should be set to `Never` if using a local image
```
Comment (algattik) [inline helm/values.yaml]: is there a reason not to use cached images?

File: helm/templates/statefulset.yaml (hunk 3 above)
Comment (ouphi) [reply helm/templates/statefulset.yaml:36]: I kept the same config as the one specified in the README of the observability extension.

File: helm/templates/statefulset.yaml (hunk 1 above)
Comment (ouphi) [reply helm/templates/statefulset.yaml]: Ah thanks 👍 !

File: helm/values.yaml (hunk 4 above)
Comment (ouphi) [reply helm/values.yaml]: It made sense to me as I was doing my test with always the same tag.
But yes makes more sense to use the cached image. Will update it to IfNotPresent.

**Pull request closed without merging**